.travis.yml

```diff
@@ -69,6 +70,9 @@ install:
 
     if [ ! -z "$TRACE_CODE_COVERAGE" ] ; then
         FFCONFIG="$FFCONFIG --enable-code-coverage --enable-debug"
+        if ! curl --head --fail --silent "https://coveralls.io/github/${TRAVIS_REPO_SLUG}" > /dev/null ; then
+            COVERALLS_IS_SET_UP=false
```

Comment:
I think you could just

```bash
unset TRACE_CODE_COVERAGE
```

and not have `COVERALLS_IS_SET_UP` at all.

Reply:
Actually thinking about this a bit more, you could perhaps have the coveralls build as an allowable failure via the `allow_failures` setting, and just exit early in this case

Reply:
hi @jtanx, I discarded `allow_failures` because using it would mean that failures run the coverage script or to upload to coveralls even from the official repo/branch would be "ignored", something I'd like to avoid.

But yes, using `allow_failures` would save us some line compared to this solution.

Reply:
I think it's somewhat saved by the fact that if nothing gets uploaded to coverity, then eventually it will register the coverage as being 0%, which marks the build as a failure. 

In any case, you shouldn't need the extra `COVERALLS_IS_SET_UP` variable.